include/hdr/hdr_histogram.h

```diff
@@ -31,8 +33,10 @@ struct hdr_histogram
     double conversion_ratio;
     int32_t counts_len;
     int64_t total_count;
-    int64_t* counts;
-};
+
```

Comment:
Can you please add a descriptive comment why we need to make this as flexible array. Also note in the comment any additional fields should be added above it.

Reply:
will do.